Exercises/Modules/Chapter 06/Soln6_06.cpp

```diff
@@ -0,0 +1,26 @@
+// Exercise 6-6. Storing a dynamically allocated vector in a smart pointer
+
+import <iostream>;
+import <format>;
+#include <cmath>
```

Comment:
I saw this on a couple of places, and it's not important, but I always group imports and #include's.